spec/graphql/mutations/create_new_user_spec.rb

```diff
@@ -27,8 +27,8 @@ def query
           user: createUser(
           authProvider: {
             credentials: {
-              username: 'Gertie'
-              password: 'password'
+              username: "Gertie",
```

Comment:
Are these hard coded right now? If I get a successful query will I see this as the user?